libs/fonts/freetype/stone.yml

```diff
@@ -0,0 +1,30 @@
+#
+# SPDX-FileCopyrightText: © 2020-2022 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : freetype
+version     : '2.13.0'
+release     : 1
+homepage    : https://freetype.org
+upstreams   :
+    - https://sourceforge.net/projects/freetype/files/freetype2/2.13.0/freetype-2.13.0.tar.xz/download : 5ee23abd047636c24b2d43c6625dcafc66661d1aca64dec9e0d05df29592624c
+summary     : Library to rasterize glyphs from font files
+description : |
+    freetype is written in C, designed to be small, efficient, highly customizable, and portable while capable of producing high-quality output (glyph images) of most vector and bitmap font formats.
+license     :
+    - FTL OR GPL-2.0-only
+builddeps   :
+    # NOTE: freetype has an optional dep on harfbuzz which
```

Comment:
I don't think we need comments like this in the packaging to be honest?
These should be in the commit message. Our digraph should be able to
break circular dependencies automatically, and we 100% do want harfbuzz
enabled =)